client/src/components/Header/components/NavLinks.js

```diff
@@ -1,29 +1,39 @@
 import React from 'react';
 import { Link } from '../../helpers';
-
 import PropTypes from 'prop-types';
 
+import Login from '../components/Login';
+import { avatarRenderer } from '../../../utils/avatarRenderer';
+
 const propTypes = {
-  displayMenu: PropTypes.bool
+  displayMenu: PropTypes.bool,
+  pathName: PropTypes.string.isRequired,
+  user: PropTypes.object
 };
 
-function NavLinks({ displayMenu }) {
+export function secondLink(user, pathName) {
```

Comment:
`secondLink` to me means there is a `firstLink` too? What does is aim to signify? Could it be something apt?

Reply:
the first link is the curriculum (/learn) that does not change. the second link is a conditional sign in/portfolio/avatar link.

I will make it more descriptive.

Reply:
I would just call it `authOrUserLink`, note we would have more links on the top nav with coderadio and others?